tests/test_parser.py

```diff
@@ -1000,8 +1003,21 @@ def test_expansion_limit_word(self):
         )
 
     def test_command_arithmetic(self):
-        self.assertRaisesRegex(NotImplementedError, 'arithmetic expansion',
-                                parse, 'a "$((2 + 2))"')
+      s= 'a $((2 + 2))'
```

Comment:
It looks like the spacing here is wrong when compared to the rest of the file, can you set it to 4?